packages/desktop-client/src/components/table.tsx

```diff
@@ -175,7 +178,35 @@ export function UnexposedCellContent({
         textOverflow: 'ellipsis',
       }}
     >
```

Comment:
The reason adding a negative left margin to the button results in truncation is that this element wraps the `<Button>`. I think the better approach here would be to use the `unexposedContent` prop to inject the `<Button>` directly without this wrapping `<Text>` (maybe using a component that wraps `<Cell>` so that logic doesn’t need to be duplicated too much?) That would allow you to remove the extra padding inside the table. (Also, adding the `whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis'` combo to the `<Button>` content would help the text truncate more gracefully on narrow window widths)

Reply:
Thanks for the suggestions. I think we got there. Let me know!